docs/reference/query-languages/query-dsl/query-dsl-kql-query.md

```diff
@@ -0,0 +1,148 @@
+---
+navigation_title: "KQL"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-kql-query.html
+applies_to:
+  stack: preview 9.0, ga 9.1
+  serverless: all
+---
+
+# KQL query [query-dsl-kql-query]
+
+Returns documents matching a provided KQL expression. The value of the `query` parameter is parsed using the
+[Kibana Query Language (KQL)](docs-content://query-languages/kql.md) and rewritten into standard Query DSL.
```

Comment:
```suggestion
[Kibana Query Language (KQL)](/reference/query-languages/kql.md) and rewritten into standard Query DSL.
```

Reply:
there's a few instances of these 

not sure why CI isn't flagging these links

Reply:
I have updated them all. Will check the preview once the CI will have finished to run.